site/source/docs/api_reference/fetch.rst

```diff
@@ -389,7 +389,7 @@ be dealt with separately.
     strcpy(attr.requestMethod, "GET");
     attr.attributes = EMSCRIPTEN_FETCH_LOAD_TO_MEMORY;
     // Make a Range request to only fetch bytes 10 to 20
-    const char* headers[] = {"Range", "bytes=10-20"};
+    const char* headers[] = {"Range", "bytes=10-20", nullptr};
```

Comment:
Since this seems to be C sample how about "NULL"?

Reply:
Makes sense, changed it to `NULL`.